rewrite-maven/src/main/java/org/openrewrite/maven/AddPlugin.java

```diff
@@ -154,13 +169,13 @@ public Xml.Tag visitTag(Xml.Tag tag, ExecutionContext ctx) {
                     }
                 } else {
                     Xml.Tag pluginTag = Xml.Tag.build("<plugin>\n" +
-                            "<groupId>" + groupId + "</groupId>\n" +
-                            "<artifactId>" + artifactId + "</artifactId>\n" +
-                            (version != null ? "<version>" + version + "</version>\n" : "") +
-                            (executions != null ? executions.trim() + "\n" : "") +
-                            (configuration != null ? configuration.trim() + "\n" : "") +
-                            (dependencies != null ? dependencies.trim() + "\n" : "") +
-                            "</plugin>");
+                                                      "<groupId>" + groupId + "</groupId>\n" +
+                                                      "<artifactId>" + artifactId + "</artifactId>\n" +
+                                                      (version != null ? "<version>" + version + "</version>\n" : "") +
+                                                      (executions != null ? executions.trim() + "\n" : "") +
+                                                      (configuration != null ? configuration.trim() + "\n" : "") +
+                                                      (dependencies != null ? dependencies.trim() + "\n" : "") +
+                                                      "</plugin>");
```

Comment:
Let's minimize these types of changes.

Reply:
I'll revert the indent changes!

Reply:
I think if you put the initial `"<plugin>"` on a new line as well you stop IntelliJ from butchering the indentation.